scala/scala_toolchain.bzl

```diff
@@ -137,6 +138,10 @@ scala_toolchain = rule(
             default = True,
             doc = "Enable writing of statsfile",
         ),
+        "use_argument_file_in_runner_for_improved_performance": attr.bool(
+            default = False,
+            doc = "Changes java binaries scripts (including tests) to use argument files and not classpath jars to improve performance, requires java > 8",
```

Comment:
Did you mean JDK, with `java < 8`?

Reply:
I actually meant the JRE (e.g. the jvm).
Do you want me to change the phrasing? I can also write JDK though it's the JRE part of it that we care about. I don't remember if in the current state of affairs in bazel these concepts are bundled or not